app.py

```diff
@@ -0,0 +1,36 @@
+import manager
+from flask import Flask
+from flask import request
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+
+
+@app.route('/init')
+def init():
+    manager.init()
+    return "ok"
+
+
+@app.route('/index')
+def index():
```

Comment:
that index all of the documents, but we need an endpoint to index one document

Reply:
Added api for index single document.